src/ui.cpp

```diff
@@ -532,20 +529,21 @@ UpdateDialog::UpdateDialog()
     m_updateButtonsSizer = new wxBoxSizer(wxHORIZONTAL);
     m_updateButtonsSizer->Add
                           (
-                            new wxButton(this, ID_SKIP_VERSION, _("Skip this version")),
+                            new wxButton(this, ID_REMIND_HOUR, _("Remind me in an hour")),
                             wxSizerFlags().Border(wxRIGHT, PX(20))
                           );
-    m_updateButtonsSizer->AddStretchSpacer(1);
     m_updateButtonsSizer->Add
                           (
-                            new wxButton(this, ID_REMIND_LATER, _("Remind me later")),
+                            new wxButton(this, ID_REMIND_TOMORROW, _("Remind me tomorrow")),
                             wxSizerFlags().Border(wxRIGHT, PX(10))
                           );
-    m_updateButtonsSizer->Add
+	m_updateButtonsSizer->AddStretchSpacer(1);
+	m_updateButtonsSizer->Add
                           (
-                            m_installButton = new wxButton(this, ID_INSTALL, _("Install update")),
+                            m_installButton = new wxButton(this, ID_INSTALL, _("Download")),
```

Comment:
Just for consistency, wouldn't it be better to rename ID_INSTALL to ID_DOWNLOAD?